test/emscripten

```diff
@@ -1 +1 @@
-Subproject commit 1d979a75722a513b586e6705d662ff4ee0ea832b
```

Comment:
This is a downgrade of the emscripten version. I'm guessing your local client just isn't up to date? (Updated it with `git submodule update`).